source/class/qx/tool/cli/Cli.js

```diff
@@ -360,8 +360,10 @@ qx.Class.define("qx.tool.cli.Cli", {
           config.packages[library.uri] = library.path;
         });
       }
-
-      if (config.libraries) {
+      // check if we need to load libraries, needs more robust test
+      let needLibraries = qx.lang.Type.isArray(this.argv._) && this.argv._[0] !== "clean";
```

Comment:
upgrade do not need loading libraries either

Reply:
True!

Reply:
Can you add that in your PR?